src/i18n/Data/Sources.php

```diff
@@ -91,23 +91,23 @@ public function getLangDirs()
         }
         $paths = [];
 
-        // Search sorted modules
+        // Search sorted modules (receives absolute paths)
         foreach ($this->getSortedModules() as $module => $path) {
             $langPath = "{$path}/lang/";
             if (is_dir($langPath)) {
                 $paths[] = $langPath;
             }
         }
 
-        // Search theme dirs
+        // Search theme dirs (receives relative paths)
         $locator = ThemeResourceLoader::instance();
         foreach (SSViewer::get_themes() as $theme) {
             if ($locator->getSet($theme)) {
                 continue;
             }
             $path = $locator->getPath($theme);
-            $langPath = "{$path}/lang/";
-            if (is_dir($langPath)) {
+            $langPath = BASE_PATH . "/{$path}/lang/";
+            if ($langPath) {
```

Comment:
Oh my, yes. Illustrates the danger of lacking test coverage to directly validate a fix against - although thankfully it seems like other tests using i18n have caught this implicitly. `vendor/bin/phpunit framework/tests/php/i18n/` is passing for me locally now, updated the commit.